pkg/config/validation.go

```diff
@@ -0,0 +1,31 @@
+package config
+
+import (
+	"fmt"
+
+	configapi "sigs.k8s.io/kueue/apis/config/v1beta1"
+)
+
+var (
+	errInvalidMaxValue              = fmt.Errorf("maximum value for QueueVisibility.ClusterQueues.MaxCount must be %d", queueVisibilityClusterQueuesMaxValue)
+	errInvalidUpdateIntervalSeconds = fmt.Errorf("minimum value for QueueVisibility.UpdateIntervalSeconds must be %d", queueVisibilityClusterQueuesUpdateIntervalSeconds)
+)
+
+const (
+	queueVisibilityClusterQueuesMaxValue              = 4000
+	queueVisibilityClusterQueuesUpdateIntervalSeconds = 1
+)
+
+func validate(cfg *configapi.Configuration) error {
```

Comment:
```suggestion
func validate(cfg *configapi.Configuration) field.ErrorList {
```

When doing validation, always return all errors, instead of stopping at the first encountered one.